app/assets/javascripts/controllers/middleware_server/middleware_server_controller.js

```diff
@@ -92,6 +92,37 @@ function MwServerController($scope, miqService) {
     miqService.sparkleOn();
     $scope.$broadcast('mwAddDeploymentEvent', $scope.deployAddModel);
   };
+
+  /////////////////////////////////////////////////////////////////////////
+  // Add JDBC Driver
+  /////////////////////////////////////////////////////////////////////////
+
+  $scope.jdbcDriverModel = {};
+  $scope.jdbcDriverModel.serverId = angular.element('#server_id').val();
```

Comment:
Yeah, this was before `$scope.resetJdbcDriverForm` existed. I think it can be removed altogether, as the `$scope.showJdbcDriverListener` already calls it.